src/components/landing/CreateAnnouncementEntry.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import SpeakerIcon from "@/icons/SpeakerIcon";
+import ButtonPrimary1 from "../shared/ButtonPrimary1";
+
+export default function CreateAnnouncementEntry() {
+  return (
+    <div className="flex flex-col p-6 items-start gap-6 rounded-lg bg-neutral-white">
+      <div className="flex items-center gap-4 self-stretch">
+        <SpeakerIcon className="min-w-[32px] w-[32px] h-[32px] md:w-[40px] md:h-[40px] " />
```

Comment:
- You can remove the min width since this icon doesn't scale.
- You have an extra space at the end. 
- Add `fill-neutral-800` to the icon.